panel.py

```diff
@@ -1,21 +1,18 @@
 from tkinter import *
 import tkinter.ttk as ttk
 import tkinter.font as font
-from timeline import *
+import time
 
 height = 500
-width = 1625
+width = 1500
 
 class Panel:
 
         def __init__(self,w,height,width):
-                self.week = 1
-                self.t = Timeline()
-                self.t.add_week()
```

Comment:
This is the comment : was this supposed to be deleted in the last version of your push ?